### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by mineme0110**

### CODE REVIEW

**Review by FabioPinheiro — APPROVED** [review]

**Comment by mkbreuningIOHK** [comment]
@amagyar-iohk , could you review? Should Todor be added?

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -2,7 +2,7 @@
 * @FabioPinheiro @mineme0110
 
 # E2E tests:
-/tests/ @antonbaliasnikov
+/tests/ @amagyar-iohk
```
Comment (mkbreuningIOHK) [inline .github/CODEOWNERS:5]: Use @input-output-hk/atala-qa team instead?

Comment (mkbreuningIOHK) [reply .github/CODEOWNERS:5]: @amagyar-iohk confirmed it is preferred not to use Atala-qa

**Review by mkbreuningIOHK — APPROVED** [review]

**Review by amagyar-iohk — APPROVED** [review]
👍

**Pull request merged**